scripts/release/go_update/run_GO_Update.pl

```diff
@@ -0,0 +1,92 @@
+#!/usr/local/bin/perl  -w
+use strict;
+
+use lib '/usr/local/gkb/modules';
+use GKB::Config;
+
+use autodie qw/:all/;
+use Cwd;
+use Getopt::Long;
+use Data::Dumper;
+use File::Basename;
+use List::MoreUtils qw/uniq/;
+
+use Log::Log4perl qw/get_logger/;
+
+Log::Log4perl->init(\$LOG_CONF);
+my $logger = get_logger(__PACKAGE__);
+
+my $data_release_pipeline_application_version = "1.0.0";
+my $data_release_pipeline_application = "go-update";
+my $data_release_pipeline_tag = "go-update-1.0.0";
+my $data_release_pipeline_repository = "https://github.com/reactome/data-release-pipeline";
+my $resource_dir = "data-release-pipeline/".$data_release_pipeline_application."/src/main/resources/";
+my $start_directory = cwd(); # abs_path($0);
+
+my $release_version = @args[0];
+
+print "start_directory: ".$start_directory."\n";
+
+if (-e "data-release-pipeline") {
+  $logger->info("data-release-pipeline exists, pulling");
+  # Start on develop. Pull will fail if we're not already on a branch
+  chdir "data-release-pipeline";
+  system("git checkout develop");
+  if (!$data_release_pipeline_tag) {
+    system("git pull");
+  # if there is a tag defined, we should check that out.
+  } else {
+    #chdir "data-release-pipeline";
```

Comment:
This should be removed since the `chdir` happens after the if/else block

Reply:
It's commented out  , but I can do the cleanup later. 

This script won't be running for R69, since there won't be time for everyone to finish reviewing the Java GO Update changes (I'm still working on them)...

Reply:
Makes sense, I saw a few other ones commented out but this one seems placed in such a way that if it were uncommented inadvertently it might affect things